src/v/config/configuration.cc

```diff
@@ -1741,6 +1741,14 @@ configuration::configuration()
       "Number of partitions that can be reassigned at once",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       50)
+  , partition_autobalancing_tick_moves_drop_threshold(
+      *this,
+      "partition_autobalancing_tick_moves_drop_threshold",
+      "If the number of in progress moves drops by this threshold, a new tick "
+      "is scheduled immediately.",
```

Comment:
This comment is unclear, the parameter is a ratio, but the comment doesn't mention that, i think it should state what are the values that are divided to obtain the ratio that the threshold applies to

Reply:
reworded it.